pep-9999.rst

```diff
@@ -0,0 +1,157 @@
+
+
+Fields
+======
+
+The canonical source for the names and semantics of each of the supported
+metadata fields is the `Core Metadata Specification`_ document.
+
+The details for the update to the `Core Metadata Specification`_ document are 
+in this `Core Metadata Specification pull request`_.
+
+
+New in Version 2.2
+------------------
+
+License-Expression (optional)
+:::::::::::::::::::::::::::::
+
+A string containing a license expression using the SPDX license expression
+syntax and SPDX-listed license identifiers so that user can express accurately
+the licensing of a package in a way that is both easy to read for humans and
+easy to process automatically by tools.
+
+A string containing a license expression using `SPDX license expression syntax and
+identifiers <https://spdx.org/ids>`__ that indicates the license(s) covering the
+distribution. The license expressions are specified in the `SPDX specification 
+<https://spdx.org/spdx-specification-21-web-version#h.jxpfx0ykyb60>`_.
+SPDX also publishes a companion `list of license ids <https://spdx.org/license-list>`_
+for use in license expressions. 
+
+Note: this field is not case sensitive though the normalized representation of
+an SPDX license expression has a well defined case.
+
+Examples::
+
+    License-Expression: BSD-3-Clause
+
+    License-Expression: MIT OR GPL-2.0-or-later AND (FSFUL AND BSD-2-Clause)
+
+    License-Expression: GPL-3.0-or-later WITH Classpath-Exception-2.0 OR BSD-3-Clause
+
+
+Tools `should` validate that this field contains a valid license expression
+and `should` report a warning if the field contains an invalid expression.
+
+
+Changed in Version 2.2
+----------------------
+
+License
+:::::::
+
+This field will be deprecated in the future and its documentation has been
+updated accordingly. Tools are encouraged to provide a warning when this field
+is used.
+
+Classifiers
+:::::::::::
+
+The use of license-related classifiers in this field will be deprecated in the
+future and its documentation has been updated accordingly. Tools are encouraged
+to provide a warning when this field is used with license-related classifiers.
+
+
+Summary of Differences From PEP 566
+===================================
+
+* Metadata-Version is now 2.2.
+
+* Added one new field: ``License-Expression``
+
+* Updated the documentation of two fields:  ``License`` and ``Classifiers``
+
+
+References
+==========
+
+This document specifies version 2.2 of the metadata format.
+Version 1.0 is specified in PEP 241.
+Version 1.1 is specified in PEP 314.
+Version 1.2 is specified in PEP 345.
+Version 2.0, while not formally accepted, was specified in PEP 426.
+Version 2.1 is specified in PEP 566.
+
+.. _`Core Metadata Specification`:
+   https://packaging.python.org/specifications/core-metadata/
+
+.. _`Core Metadata Specification pull request`:
+   https://github.com/pypa/packaging.python.org/pull/635/
+
+
+Copyright
+=========
+
+This document has been placed in the public domain.
```

Comment:
Hello @pombredanne,

The requirements at [PEP 1](https://www.python.org/dev/peps/pep-0001/#what-belongs-in-a-successful-pep) was changed recently in #1117, so can you please update the `Copyright` section to make this PEP meet them?
> 13. Copyright/license -- Each new PEP must be placed under a dual license of public domain and CC0-1.0-Universal [8] (see this PEP for an example).

```suggestion
This document is placed in the public domain or under the CC0-1.0-Universal license, whichever is more permissive.
```

Reply:
@Jamim this is done. Thank you!